beginner/apps/src/bin/radio-puzzle-2.rs

```diff
@@ -6,16 +6,18 @@ use cortex_m_rt::entry;
 // NOTE you can use `FnvIndexMap` instead of `LinearMap`; the former may have better
 // lookup performance when the dictionary contains a large number of items but performance is
 // not important for this exercise
-use heapless::{consts, LinearMap};
-use panic_log as _; // the panicking behavior
+use heapless::LinearMap;
+// this imports `beginner/apps/lib.rs` to retrieve our global logger + panicking-behavior
+use apps as _;
 
 #[entry]
 fn main() -> ! {
     dk::init().unwrap();
 
     // a dictionary with capacity for 2 elements
-    let mut dict = LinearMap::<_, _, consts::U2>::new();
-    //                                       ^^ capacity; this is a type not a value
+    let mut dict = LinearMap::<_, _, 2>::new();
+    //                          content types ^^ ^^  ^ capacity
```

Comment:
these markers (in the comment) no longer match the preceding line of code -- this happens in other files as well

Reply:
ah dangit. they only match if you're using Rust-Analyzer. I wonder how to best resolve this

Reply:
the original assumes you are not viewing the code with Rust-Analyzer's annotations so perhaps we can leave it like that? The other option I can think of is discarding the marker and instead describing the type parameters with text in a *preceding* comment (first param is X, second is Y, etc.)